qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -177,6 +180,30 @@ public void test70BindMountCustomPathConfAndJvmOptions() throws Exception {
         assertThat(nodesResponse, containsString("\"using_compressed_ordinary_object_pointers\":\"false\""));
     }
 
+    /**
+     * Check that the default config can be overridden using a bind mount, and that env vars are respected
+     */
+    public void test71BindMountCustomPathWithDifferentUID() throws Exception {
+        copyFromContainer(installation.config("elasticsearch.yml"), tempDir.resolve("elasticsearch.yml"));
```

Comment:
I don't understand why this file is being copied. I don't see it being modified or used?

Reply:
Left over. Removed in d89f795d61b9e645bb903aa9c2693ec18838b18e